script.js

```diff
@@ -47,4 +50,119 @@ buttonElement.addEventListener('click', function (event) {
     subElement.classList.toggle('active');
 });
 
+buttonRight.addEventListener('click', function (){
+    move('right');
+}); 
+buttonLeft.addEventListener('click', function (){
+    move('left');
+});
+
+let images = document.querySelectorAll('.carousel object');
+let slider = [];
+for (let i = 0; i < images.length; i++) {
+    slider[i] = images[i].data;
+    images[i].remove();
+}
+let boxes = document.querySelectorAll('.carousel__box');
+for (let i = 0; i < boxes.length; i++) {
+    boxes[i].remove();
+}
+
+let carouselTextObject = document.createElement('div');
+carouselTextObject = carouselText;
+
+let step = 0;
+let carousel =  Array.from(document.querySelectorAll('.carousel__box'));
 
+function carouselObject (position, left) {
+    let objectDiv = document.createElement('div');
+    let objectImg = document.createElement('object');
+    objectImg.data = slider[position];
+    objectImg.classList.add('carousel__object');
+    objectDiv.classList.add('carousel__box');
+    objectDiv.style.left = left;
+    if (position === 0) {
+        objectDiv.appendChild(carouselTextObject);
+    }
+    objectDiv.appendChild(objectImg);
+    document.querySelector('.carousel').appendChild(objectDiv);
+    carousel.push(objectDiv);
+}
+
+function draw() {
+    carouselObject (0, "0%");
+    carouselObject (1, "100%");
+    carouselObject (2, "-100%");
```

Comment:
What if we have 4 or 5 images?

Reply:
We need only 3 slides here - 0, 1 and last.  If we have more slides, I can only change number 2 on number of the last element of array (it is carousel.length - 1).
Changed it that way: carouselObject (carousel.length - 1, "-100%");